outermysite/polls/imdbScraper.py

```diff
@@ -0,0 +1,118 @@
+import logging
+import sys
+import traceback
+
+import requests
+
+from bs4 import BeautifulSoup
+
+
+def get_search_results(movie):
+
+    url = "http://www.imdb.com/find?s=tt&q=" + movie
+
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        logging.info(e)
+        traceback.print_exc(e)
+
+    soup = BeautifulSoup(r.content, "html.parser")
+    movies = soup.find_all("td", {"class": "result_text"})
+
+    json_array = []
+
+    if len(movies) == 0:
+        json_object = {}
+        json_object['Error'] = "No results found"
+        return json_object
+
+    for item in movies:
+        if item is not None:
+            title = item.get_text().strip()
+            url = item.find("a").get("href")
+            movie_id = url[7:16]
+        url = "http://www.imdb.com" + url
+        json_object = {}
+        json_object['Title'] = title
```

Comment:
JSON keys should be in camel case.
https://google.github.io/styleguide/jsoncstyleguide.xml#Property_Name_Guidelines